docs/migration-guides/1.22-1.23.md

```diff
@@ -0,0 +1,23 @@
+# Migrate from 1.22 to 1.23
+
+## Increase the Loki volume size
+In Kyma 1.23.0, the Loki volume size requested by the persistent volume claim has been increased from 10Gi to 30Gi
+
+## Upgrade process
+During the upgrade process, the pre-upgrade job is patching the volume size so as to increase the size from 10Gi to 30Gi as the retention time for logs has been increased from 1 day to 5 days. However, the upgrade process would succeed if the field `allowVolumeExpansion` is set to `true` for the storage class. Please look [here](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims) for more info.
+
+If the flag is not set on the storage class then the size of the volume would not change. Use following command to check the size of the volume
+
+```
+kubectl get persistentvolumeclaims -n kyma-system storage-logging-loki-0
+```
+if the size is still the old size follow the steps below and then trigger update
+
+```
+kubectl delete statefulsets.apps -n kyma-system logging-loki
+
+kubectl delete persistentvolumeclaims -n kyma-system storage-logging-loki-0
+```
+> This action will delete the existing data.
+
+If there is custom volume size is set for the loki persistent volume claim, then evaluate if the current size would suffice for the logs for retention time of 5 days.
```

Comment:
```suggestion
Make sure that your custom volume size for the Loki persistent volume claim is sufficient for the new retention time of 5 days.
```

I put a similar hint right at the top, so I'm not sure if this is needed here again. Not sure if we want users to evaluate that right away, or at the end.